benchmarks/pipe.py

```diff
@@ -25,7 +25,7 @@
 from fairscale.nn import Pipe
```

Comment:
In this script we are using model.group check in the `log_number_of_parameters` function (which i saw is a reason for a test failure). We'll have to modify that to check for Pipelinestyle instead of group.